pkg/clusterlogging/clusterlogging.go

```diff
@@ -0,0 +1,190 @@
+package clusterlogging
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	clov1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ClusterLoggingBuilder provides struct for clusterLogging object.
+type ClusterLoggingBuilder struct {
```

Comment:
This is the only 1 object under clusterlogging package. 
Please re-name it to Builder.